### ISSUE TITLE
Add in the readme the browser extension

### ISSUE THREAD

**Issue by Mte90** [mozilla/togetherjs#1115]
The repo is https://github.com/Mte90/TogetherJS-Companion
I am waiting the approval in the chrome and firefox addons marketplace :-)

**Comment by Romainpetit** [comment]
@Mte90 It looks like it's online now ! https://chrome.google.com/webstore/detail/togetherjs-companion/amfgekclmjcabbfjildgdhjcoghcidnl?hl=en
Very empty though ( screenshots, logo ... )

**Comment by Mte90** [comment]
Yes, I have to update the screenshot, but on Firefox was rejected because I inject the js file from the online together.js website, so I have to release an update but I am busy at the moment, but is on my queue!

**Comment by woniesong92** [comment]
Oh wow! This looks cool. Do you still maintain it?

**Comment by Mte90** [comment]
https://addons.mozilla.org/it/firefox/addon/togetherjs-companion/
Released also on firefox :-)
I mantain the extension of course :-)
